recipes/vs2008_runtime/bld.bat

```diff
@@ -0,0 +1,25 @@
+if "%ARCH%" == "32" (set "ARCH_DIR=x86")
+if "%ARCH%" == "64" (set "ARCH_DIR=amd64")
+
+dir /a:d /o:-n /s /b C:\Windows\WinSxS\%ARCH_DIR%_microsoft.vc90.openmp* > vcomp90_locs.txt
+type vcomp90_locs.txt
+
+set /p VCOMP_DIR=<vcomp90_locs.txt
```

Comment:
Extract the first entry from the file. Should include the latest version for the target architecture.